src/js/Node.js

```diff
@@ -9,6 +9,44 @@ define(['./appendNodeFactory', './util'], function (appendNodeFactory, util) {
       return obj === Object(obj);
   }
 
+  function _pathIsInReadOnlyPaths(aimaraPath, readOnlyAimaraPaths) {
+    var currentPath, areEquals;
+
+    for (var iPaths = 0, lPaths=readOnlyAimaraPaths.length; iPaths < lPaths; iPaths++) {
+      currentPath = readOnlyAimaraPaths[iPaths];
+      areEquals = false;
+
+      if (aimaraPath.length == currentPath.length) {
+        areEquals = true;
+        for (var iCurrentPath = 0, lCurrentPath=currentPath.length; iCurrentPath < lCurrentPath; iCurrentPath++) {
+          if (aimaraPath[iCurrentPath] != currentPath[iCurrentPath]) { 
+            areEquals = false;   
+            iCurrentPath = lCurrentPath;
+          }           
+        }       
+      }
+
+      if (areEquals) {
+        return true;
+      }
```

Comment:
I don't like it neither, but since requirements like that must be bundled using browserify (because of node's require), jsoneditor.js would end up being a giant .js like the others we are using. I thought it would be better to just write that logic.
Didn't know of Array.join, agree it would be far better to use it instead of all that code.